### PR TITLE
improve 'METHOD_DENIED' error message

### PR DESCRIPTION
**Pull request by luclu**
Test via `menu -> develop -> run tests`.

Also removes unnecessary `Window` object.

### CODE REVIEW

**Comment by evertonfraga** [comment]
I saw some user log with a possible error on the error handling. `__method__` wasn't replaced.

```
[2017-04-11 10:11:41.421] [ERROR] ipcProviderBackend - Send request failed { code: -32601, message: 'Method \'__method__\' not allowed.' }
```

https://gitter.im/ethereum/mist?at=58ec5ffb8fcce56b20edea47

File: modules/ipc/methods/base.js [hunk 1]
```diff
@@ -97,11 +97,11 @@ module.exports = class BaseProcessor {
         }
 
         // prevent dapps from acccesing admin endpoints
-        if (!/^eth_|^shh_|^net_|^web3_|^db_/.test(payload.method)) {
+        if (!/^eth_|^bzz_|^shh_|^net_|^web3_|^db_/.test(payload.method)) {
             delete payload.result;
-
-            payload.error = this.ERRORS.METHOD_DENIED;
+            const err = this.ERRORS.METHOD_DENIED;
+            err.message = err.message.replace(/__method__/, payload.method);
```
Comment (frozeman) [inline modules/ipc/methods/base.js]: i guess a simple string replace `'__method__'` would be enough here.

Comment (frozeman) [inline modules/ipc/methods/base.js]: Are we sure we are not modifying the original object?

**Comment by luclu** [comment]
Thanks @frozeman!

**Pull request merged**